### DISCUSSION TITLE
2023/12/11/snapdragon-410-portable-wifi-hotspot-flash-debian-and-optimize/

### DISCUSSION THREAD

**Discussion by giscus** [hmsjy2017/blog-comment#9] [Announcements]
# 2023/12/11/snapdragon-410-portable-wifi-hotspot-flash-debian-and-optimize/

前言最近我在网上看到一款骁龙 410 SoC 的随身 WiFi，性价比高，领券后十元以下就能买到。更重要的是，这款 SoC 的随身 WiFi 能刷 Debian 系统。你可以在上面搭建网站甚至 MC 服务器，也可以接上扩展版当 Linux 开发板用，可玩性很高。 本文记录一下刷机的步骤和注意事项。 硬件配置这款随身 WiFi 配置如下：    SoC 骁龙410（MSM8916）四核 Cortex

https://blog.iamsjy.com/2023/12/11/snapdragon-410-portable-wifi-hotspot-flash-debian-and-optimize/

**Comment by MyGodKnow** [comment]
dalao请问我在刷了debian后连接了WIFI，但是WIFI棒子与WIFI的连接老是断开，请问这是怎么一回事

**Reply by hmsjy2017** [reply]
板子是什么型号的？是接在电脑USB接口还是手机充电器上？

**Comment by kalvinses** [comment]
版本很不错，但是怎么安装DbusSmsForward转发短信呢？安装一直失败。酷安有个版本固件的可以用一键命令安装【python3 -c "$(curl -sSL https://pub-f840439886cb4ca4871861e7eb5820c6.r2.dev/sms-urwo/sms.py)"】，这个版本安装就提示各种错误，一开始以为是python3依赖不全，想着重新安装一下就好了，结果是python3安装各种报错，折腾了一晚上放弃了

**Reply by hmsjy2017** [reply]
短信转发用自带的安卓配合 [SmsForwarder](https://github.com/pppscn/SmsForwarder) 更方便，Linux 下有点麻烦我也试了很久没成功

**Reply by kalvinses** [reply]
我的安卓系统版本有bug，连上wifi重启后会自动关闭然后开启热点，

**Reply by hmsjy2017** [reply]
看了下酷安，开机自动开热点应该是 App 控制的。我不知道你用的是是什么板子，如果是帝旭 UZ801，卸载 HelloJetty 和 HelloSms 就不会自动开热点了。

参考：https://www.coolapk.com/feed/60301630?shareKey=MTVmMWExZTk3MzRjNjc0Yzk4NDQ

**Reply by kalvinses** [reply]
虽然我的板子是UFI001c，但还是非常感谢您的回复，我刷回去试试看是不是其他软件控制了。另外我用GPT找到这个固件不能启动短信转发的原因了，可能是因为这个固件内核版本是自编译过的，缺少qmi_wwan 模块，所以安装Dbus会失败。而且因为是自编译的，也不能单独安装这个模块。再次感谢热心回复

**Reply by yyx248131667** [reply]
https://github.com/lkiuyu/DbusSmsForward
https://github.com/3899/SimAdmin这两个都支持短信转发

**Comment by maq917** [comment]
按教程一步一步的走成功了，但是我遇到个问题就是不能识别sim卡，一直提示sim-missing，有什么办法处理吗？

**Reply by hmsjy2017** [reply]
刷之前要先切卡，默认卡槽是 eSIM

**Reply by maq917** [reply]
> 刷之前要先切卡，默认卡槽是 eSIM

已经刷了的怎么救，没有miko备份，只有9008Tool的备份

**Reply by maq917** [reply]
miko备份报错，就没去管。。

**Reply by hmsjy2017** [reply]
你看能不能把备份刷回去再切卡

**Reply by maq917** [reply]
> 你看能不能把备份刷回去再切卡

9008Tool的备份还没没法刷debian了，img拖不进去了
